model/delivery/LtiTestTakerAuthorizationService.php

```diff
@@ -15,33 +15,36 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2017 (original work) Open Assessment Technologies SA;
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
  *
  */
 
 namespace oat\ltiProctoring\model\delivery;
 
 use common_session_Session;
 use oat\oatbox\session\SessionService;
+use oat\taoDelivery\model\execution\DeliveryExecutionInterface;
 use oat\taoLti\models\classes\LtiException;
 use oat\taoLti\models\classes\LtiInvalidVariableException;
 use oat\taoLti\models\classes\LtiLaunchData;
 use oat\taoLti\models\classes\TaoLtiSession;
+use oat\taoLti\models\classes\user\LtiUser;
 use oat\taoProctoring\model\authorization\TestTakerAuthorizationService;
 use oat\taoDelivery\model\execution\DeliveryExecution;
 use oat\taoDelivery\model\authorization\UnAuthorizedException;
 use oat\oatbox\user\User;
 use oat\taoProctoring\model\DelegatedServiceHandler;
 use oat\taoLti\models\classes\LtiRoles;
+use oat\taoQtiTest\models\TestSessionService;
 
 /**
  * Manage the Delivery authorization.
  * @author Aleh Hutnikau, <hutnikau@1pt.com>
  */
 class LtiTestTakerAuthorizationService extends TestTakerAuthorizationService implements DelegatedServiceHandler
 {
-
-    const CUSTOM_LTI_PROCTORED = 'custom_proctored';
+    public const CUSTOM_LTI_PROCTORED = 'custom_proctored';
```

Comment:
Maybe it's not the best practice, but it is a current practice in TAO to define custom LTI params as class constants. And to keep it consistent and following [POLA principle](https://en.wikipedia.org/wiki/Principle_of_least_astonishment), I'd keep it like that. If you think it would be and improvement let's create a tech debt ticket and refactor it for other params as well